### ISSUE TITLE
Certain columns of model.summary() are cutoff in Windows Terminal or Python Shell

### ISSUE THREAD

**Issue by Moondra** [keras-team/keras#9302]
I'm trying to examine the built-in Xception model for transfer learning and used  model.summary(): 

 Here are the first few layers of the Xception model   

Layer (type)                    Output Shape         Param      Connected to                      

input_1 (InputLayer)       (None, None, None, 3 0                                             __________________________________________________________________________________________________ block1_conv1 (Conv2D)           (None, None, None, 3 864         input_1[0][0]                     __________________________________________________________________________________________________ block1_conv1_bn (BatchNormaliza (None, None, None, 3 128         block1_conv1[0][0]                ______________________________________________________________________________________

As you can see the output shape is cutoff before displaying the entire contents. 
I've tried this on Windows Terminal and Python Shell (IDLE). 
Is there a way to fix this? Thank you.
